prow/cmd/deck/main_test.go

```diff
@@ -1166,3 +1166,23 @@ func TestCanTriggerJob(t *testing.T) {
 		}
 	}
 }
+
+func TestHttpStatusForError(t *testing.T) {
+	{
```

Comment:
nit: A slice of testcase structs `{err error, expectedCode int}` and a loop would be a bit more idiomatic and less repetitive, but it doesn't really matter here.